module/src/hooks/clientside-validation/ValidationTypes.tsx

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
inconsistent casing in the filepath, should be hooks/clientsideValidation/validationTypes